apps/spark/src/main/java/com/linkedin/openhouse/jobs/scheduler/tasks/TableStatsCollectionTask.java

```diff
@@ -25,15 +25,11 @@ public JobConf.JobTypeEnum getType() {
 
   @Override
   protected List<String> getArgs() {
-    TableMetadata tableMetadata = getMetadata();
-    List<String> jobArgs =
-        Stream.of("--tableName", tableMetadata.fqtn()).collect(Collectors.toList());
-
-    return jobArgs;
+    return Stream.of("--tableName", metadata.fqtn()).collect(Collectors.toList());
   }
 
   @Override
   protected boolean shouldRun() {
-    return tablesClient.canRunTableStatsCollection(getMetadata());
+    return true;
```

Comment:
Is there a case where it's not possible to run it? Seems like it was previously doing a check whether table collection is possible.